### PR TITLE
allow overridable options

### PR DESCRIPTION
**Pull request by joelcollinsdc**
I need to use an http proxy so I need to override the typhoeus options.  Please let me know if there is a better way to accomplish this...

### CODE REVIEW

File: lib/site-inspector.rb [hunk 1]
```diff
@@ -55,6 +55,10 @@ def typhoeus_defaults
           "User-Agent" => "Mozilla/5.0 (compatible; SiteInspector/#{SiteInspector::VERSION}; +https://github.com/benbalter/site-inspector)"
         }
       }
+      unless @typhoeus_options.nil?
```
Comment (konklone) [inline lib/site-inspector.rb]: If you're okay with truthiness, I recommend `if @typhoeus_options` over `unless @typhoeus_options.nil?`.

**Comment by konklone** [comment]
:+1: Looks like a reasonable API to export for people using the Ruby lib directly.

File: lib/site-inspector.rb (hunk 1 above)
Comment (joelcollinsdc) [inline lib/site-inspector.rb]: oops.

**Comment by benbalter** [comment]
Looks good. Sorry for the delay. Thanks @joelcollinsdc!

**Pull request merged**